optax/_src/wrappers.py

```diff
@@ -379,7 +379,7 @@ def do_update(_):
       return inner.update(updates, state.inner_state, params)
 
     def reject_update(_):
-      return updates, state.inner_state
+      return jax.tree_map(jnp.zeros_like, updates), state.inner_state
```

Comment:
This proposed change breaks the intended behavior of the function. 
As per the docstring: 

>  When not calling the inner update function, the `updates` and the inner state are left untouched and just passed through.